test/Plutip.purs

```diff
@@ -169,11 +167,15 @@ import Test.Spec.Runner (defaultConfig)
 main :: Effect Unit
 main = launchAff_ do
   Utils.interpretWithConfig
-    defaultConfig { timeout = Just $ wrap 70_000.0, exit = true }
-    do
-      suite
-      UtxoDistribution.suite
-      NetworkId.suite
+    defaultConfig { timeout = Just $ Milliseconds 70_000.0, exit = true }
+    $ do
+        suite
+        UtxoDistribution.suite
+        NetworkId.suite
+  Utils.interpretWithConfig
+    defaultConfig { timeout = Just $ Milliseconds 450_000.0, exit = true }
+    $ do
+        Staking.suite
```

Comment:
This wont work, the second tests don't get run due to `exit = true` on the first set.

exit is necessary as that's how hydra knows the job succeeded/failed.

Two options:
  - `exit = false` and have interpret return the results of the tests, and after both test sets we decide if we want to exit with an error code
  - add the staking suite as a separate check to hydra